java-frontend/src/test/resources/se/MethodBehavior.java

```diff
@@ -0,0 +1,25 @@
+abstract class MethodBehavior {
+  public boolean topMethod(boolean a) {
+    boolean result = false;
+    if(a) {
+      result = bar(a);
+    } else {
+      result = foo(a);
+    }
+    qix();
+    int i = "".length();
+    return result;
+  }
+
+  private boolean foo(boolean a) {
+    return a;
+  }
+
+  private boolean bar(boolean a) {
+    return !a;
+  }
+
+  private abstract void qix();
+
+
```

Comment:
I would add another method, independent of the 4 others, to show that it will be the fifth method behavior to be extracted.